magento/scripts/patch-AC2855.php

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env php
+<?php
+
+$version = getenv('MAGENTO_VERSION');
+$is244 = substr($version, 0, 5) == '2.4.4';
+$patchLevel = (int)substr($version, 7);
+
+if (!$is244 || $patchLevel > 100) {
```

Comment:
I'm not really following this `$patchLevel` comparison. You are trying to cast a string like "2.4.4-p11" to an int. In my tests the `$patchLevel` is always `1`. Is that correct?

Reply:
Oops, meant to take out the patch level. Since it applies to all versions of the `2.4.4` package.
I'll remove it after the current pipeline finishes running.

But as for the check, we are stripping out everything up to and including the `p` character. 
So we either end with the numerical value or a empty string (which casts to 0) in the case of no patch. 

**Sandbox demonstrating it:**
https://onlinephp.io?s=bYxBC4IwGIbPDfYfPoYHByZoQaBFhwg6BAlBlwhZutSD29iml-i_lwYm1e2B93mf5VqVCiOMnJoVXFh54tpUUhhYwZmE_tyfEw_eMFXBiMMRz8ZOQC4xRjepOctK96fLDDgt1xTuGE14VkogiZYZN6YSRdRvr1qyS9LtYR8PzrG5GqsjIOCD6dntVA8W9I-9YcbyvLNjaJlO86ZWrlsJS7--dPh8Go8n&v=8.2.20